cmake/common.cmake

```diff
@@ -206,7 +207,11 @@ if("sgx" IN_LIST COMPILE_TARGETS)
             ${CRYPTO_LIBRARY}
             ${CMAKE_DL_LIBS}
             ${CMAKE_THREAD_LIBS_INIT}
-            openenclave::oehostapp
+            openenclave::oehost
+            -lc++
```

Comment:
We probably ought to have a ${USE_LIBCXX} that contains all of this.

Reply:
Good point. Done now.